gobcore/model/name_compressor.py

```diff
@@ -1,33 +1,40 @@
 """Shorten table names."""
 
 
-_CONVERSIONS = {
-    "is_bron_voor_aantekening_kadastraal_object": "bron_kad_obj",
-    "ontstaan_uit_appartementsrechtsplitsing_vve": "ontst_apprechtsplit_vve",
-    "betrokken_bij_appartementsrechtsplitsing_vve": "betr_apprechtsplit_vve",
-    "heeft_een_relatie_met": "hft_rel_mt",
-    "heeft_betrekking_op": "hft_btrk_p",
-    "is_onderdeel_van": "ondrdl_vn",
-    "aangeduid_door": "angdd_dr",
-    "wordt_uitgeoefend_in": "uitgoef_in",
-    "commerciele_vestiging": "comm_vstgng",
-    "maatschappelijke_activiteit": "maatsch_act",
-    "heeft_sbi_activiteiten": "heeft_sbi_act",
-    "refereert_aan_meetbouten_referentiepunten": "meetbouten_refpnt_rft_aan",
-    "betrokken_bij_brk_zakelijke_rechten": "betrokken_bij_brk_zrt",
-    "is_beperkt_tot_brk_tenaamstellingen": "is_beperkt_tot_brk_tng",
-    "ontstaan_uit_brk_zakelijke_rechten": "ontstaan_uit_brk_zrt",
-    "betrokken_samenwerkingsverband_brk_subject": "betr_samenwerkverband_brk_sjt",
-    "betrokken_gorzen_en_aanwassen_brk_subject": "betr_gorzen_aanwassen_brk_sjt",
-}
+from collections import OrderedDict
+
+_CONVERSIONS = OrderedDict(
```

Comment:
Heb je mss een testgeval waar OrderedDict voor nodig is?

Reply:
Ik ga even kijken.

Reply:
Ik heb _NameCompressor order tests_ toegevoegd.